From 27330c60905badb0a7b3f9297ffd851d7b312e0e Mon Sep 17 00:00:00 2001
From: Eskil Abrahamsen Blomfeldt <eskil.abrahamsen-blomfeldt@qt.io>
Date: Tue, 16 Sep 2025 13:38:16 +0200
Subject: [PATCH 044/239] vectorimage: Add dependency on QtQuick.Shapes

The generated code may (and will) depend on QtQuick.Shapes, but
this cannot be determined by the Android deploy tool without some
help. This adds a dependency on that module so that you don't have
to manually add it to your own QML code.

Task-number: QTBUG-140187
Change-Id: Id888407345d3c1932d3200d540aa7d2b56ef228f
Reviewed-by: Eirik Aavitsland <eirik.aavitsland@qt.io>
(cherry picked from commit 567c5a0b1a87b51d636c7ba2499765e537f63c71)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/quickvectorimage/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/quickvectorimage/CMakeLists.txt b/src/quickvectorimage/CMakeLists.txt
index 5c1dce4cb5..a179c13603 100644
--- a/src/quickvectorimage/CMakeLists.txt
+++ b/src/quickvectorimage/CMakeLists.txt
@@ -31,6 +31,7 @@ qt_internal_add_qml_module(QuickVectorImage
     PLUGIN_TARGET qquickvectorimageplugin
     CLASS_NAME QtQuickVectorImagePlugin
     IMPORTS QtQuick.VectorImage.Helpers
+    DEFAULT_IMPORTS QtQuick.Shapes
     SOURCES
     qquickvectorimage_p.h qquickvectorimage.cpp
     qquickvectorimage_p_p.h
-- 
2.51.2

